changelog.txt

```diff
@@ -1,5 +1,112 @@
 == Changelog ==
 
+= 5.1.0 2021-03-09 =
+
+**WooCommerce**
+
+* Update - WooCommerce Admin package 2.0.2. #29111
```

Comment:
Technically that PR increased the package to 2.0.1 and then #29229 did the 2.0.2. Do you think we should note both PRs?

Reply:
For https://github.com/woocommerce/woocommerce/pull/29093 I was thinking this was a fix to an unreleased change so I thought we don't need to include it. I could be wrong.

Reply:
I believe #29093 is implementing an enhancement request from Isotope that changes a widget URL to direct to the new analytics page instead of the deprecated one.